src/bin/pg_autoctl/pgsetup.c

```diff
@@ -327,6 +327,16 @@ pg_setup_init(PostgresSetup *pgSetup,
 				options->authMethod, NAMEDATALEN);
 	}
 
+	if (options->quorum >= 0)
```

Comment:
Do we need this test? if yes, why? I would think we want to duplicate the default value when that's the one that is setup at this point...

Reply:
they were there for additional checks, guess it is too much